homework_3/css/style.css

```diff
@@ -0,0 +1,64 @@
+body {
+	background: #282828;
+}
+
+.box {
+	background-image: url(../img/background.jpg);
```

Comment:
it`s better to get the pattern from the image in png format, set for this block black background and background image with that pattern